pkg/tests/api/correlations/correlations_update_test.go

```diff
@@ -329,9 +357,37 @@ func TestIntegrationUpdateCorrelation(t *testing.T) {
 		require.Equal(t, "Correlation updated", response.Message)
 		require.Equal(t, "2", response.Result.Label)
 		require.Equal(t, "2", response.Result.Description)
+		require.Equal(t, "field", response.Result.Config.Field)
+		require.Equal(t, map[string]interface{}{"expr": "bar"}, response.Result.Config.Target)
+		require.NoError(t, res.Body.Close())
+
+		// partially updating only config
+		res = ctx.Patch(PatchParams{
+			url:  fmt.Sprintf("/api/datasources/uid/%s/correlations/%s", correlation.SourceUID, correlation.UID),
+			user: adminUser,
+			body: `{
+				"config": {
+					"field": "name",
+					"type": "query",
+					"target": { "expr": "baz" }
+			}}`,
```

Comment:
nit: it's be nice to keep indentation consistent for readability

```suggestion
				}
			}`,
```

Reply:
True. I changed it.